files/initramfs-tools/union-mount.j2

```diff
@@ -11,8 +11,22 @@ case $1 in
     ;;
 esac
 
+# Extract kernel parameters
+set -- $(cat /proc/cmdline)
+for x in "$@"; do
```

Comment:
You could probably combine the above 2 lines into:

for x in $(cat /proc/cmdline); do

So that the $@ is not changed.

Reply:
Changed